pyproject.toml

```diff
@@ -15,8 +15,15 @@ directory = "newsfragments"
 underlines = ["-", "~", "^"]
 issue_format = "`#{issue} <https://github.com/python-trio/trio/issues/{issue}>`__"
 
-# Unfortunately there's no way to simply override
-# tool.towncrier.type.misc.showcontent
+[[tool.towncrier.type]]
+directory = "headline"
+name = "Headline features"
```

Comment:
Let's be consistent about capitalization -- either this should be "Headline Features" or (e.g.) `deprecated` should be "Deprecations and removals". (I personally prefer leading-capital-only, but title-case is fine too.)

Reply:
Agreed. And #1574 was just merged, sorry about the "conflict"